<launch>
  <param name="robot_description" textfile="$(find openarms)/urdf/oa9.xml"/>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher"/>
  <node name="oa_state" pkg="openarms" type="oa-state"/>
  <node name="rviz" pkg="rviz" type="rviz"/>
  <node name="armbus" pkg="openarms" type="armbus_sim"/>
  <node name="control_jointspace" pkg="openarms" type="control_jointspace">
    <remap from="arm_actuators_autopilot" to="arm_actuators"/>
  </node>
</launch>
